archive/imo/imo2005_q3.lean

```diff
@@ -0,0 +1,68 @@
+/-
+Copyright (c) 2021 Manuel Candales. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Manuel Candales
+-/
+import data.real.basic
+
+/-!
+# IMO 2005 Q3
+Let x, y and z be positive real numbers such that xyz ≥ 1. Prove that:
```

Comment:
Please add backticks or dollars around variables and math expressions.

Reply:
done!